### PR TITLE
Pablo/port createtileobjecttool

### PR DESCRIPTION
**Pull request by azagaya**

### CODE REVIEW

File: src/tiled/propertybrowser.cpp [hunk 1]
```diff
@@ -2033,8 +2033,9 @@ Properties PropertyBrowser::combinedProperties() const
     // Add properties from selected objects which mObject does not contain to mCombinedProperties.
     const auto currentObjects = mDocument->currentObjects();
     for (Object *obj : currentObjects) {
-        if (obj != mObject)
-            mergeProperties(combinedProperties, obj->properties());
+        if (obj == mObject || obj->wasRandomized())
```
Comment (jessetrem) [inline src/tiled/propertybrowser.cpp:2037]: Could you get these in // EDEN CHANGE blocks just to keep things consistent.

File: src/tiled/tilestamp.h [hunk 2]
```diff
@@ -46,8 +46,16 @@ struct TileStampVariation
         Q_ASSERT(map->layerAt(0)->isTileLayer());
     }
 
+    TileLayer *tileLayer() const;
```
Comment (jessetrem) [inline src/tiled/tilestamp.h:50]: // EDEN CHANGE pls

File: src/tiled/createtileobjecttool.h [hunk 3]
```diff
@@ -34,6 +36,29 @@ class CreateTileObjectTool : public CreateObjectTool
...
+    void setStamp(const TileStamp &stamp);
+
+    /**
+     * This returns the current stamp used for filling.
+     */
+    const TileStamp &stamp() const { return mStamp; }
+
+    void doUseStamp(bool bUse){ useStamp=bUse; }
+
+    /// <summary>
```
Comment (jessetrem) [inline src/tiled/createtileobjecttool.h]: Pretty sure I made this mistake, could we also get this surrounded with `// EDEN CHANGE` and get rid of the `/// <summary>`

File: src/tiled/createtileobjecttool.cpp [hunk 4]
```diff
@@ -40,6 +40,7 @@ CreateTileObjectTool::CreateTileObjectTool(QObject *parent)
     setShortcut(Qt::Key_T);
     Utils::setThemeIcon(this, "insert-image");
     languageChangedImpl();
+    useStamp = false;
```
Comment (jessetrem) [inline src/tiled/createtileobjecttool.cpp:45]: Missing // EDEN CHANGE

**Review by jessetrem — CHANGES_REQUESTED** [review]

File: src/tiled/tilestamp.h (hunk 2 above)
Comment (azagaya) [reply src/tiled/tilestamp.h:50]: I think this is unused? maybe we can just remove it?

Comment (azagaya) [reply src/tiled/tilestamp.h:50]: Just left it there to be sure

**Review by jessetrem — APPROVED** [review]

**Pull request merged**